### ISSUE TITLE
CC005 - false positives on Some conditions

### ISSUE THREAD

**Issue by DinoChiesa** [apigee/apigeelint#542]
```
error CC005 : Invalid Condition - cannot parse    
```

example
```
          <Condition>NOT((request.header.content-type =| "text/xml") OR 
                     (request.header.content-type =| "application/xml" ))</Condition>
```
